calyx/src/passes/infer_static_timing.rs

```diff
@@ -0,0 +1,71 @@
+use std::collections::HashMap;
```

Comment:
Needs a doc comment at the top. Doc comments have the form:
```
//! This is a pass
//! It does blah blah
```

Reply:
Ack!